### PR TITLE
Fix InitNornir example in docs/configuration/index.rst

### PR DESCRIPTION
**Pull request by ubaumann**
Fix "Broken example for InitNornir" from #594

### CODE REVIEW

File: docs/howto/advanced_filtering/config.yaml [hunk 1]
```diff
@@ -4,4 +4,7 @@ inventory:
     options:
         host_file: "advanced_filtering/inventory/hosts.yaml"
         group_file: "advanced_filtering/inventory/groups.yaml"
-
+runners:
```
Comment (ktbyers) [inline docs/howto/advanced_filtering/config.yaml:7]: Needs to be `runner` and not `runners`

**Comment by ktbyers** [comment]
Everything else looks good except for the aforementioned `runners` to `runner` change.

File: docs/howto/advanced_filtering/config.yaml (hunk 1 above)
Comment (ubaumann) [reply docs/howto/advanced_filtering/config.yaml]: Thank you for seeing it!

**Comment by ktbyers** [comment]
FYI, the testing failures are due to other issues which I fixed in a separate PR.

**Review by dbarrosop — APPROVED** [review]

**Comment by dbarrosop** [comment]
would you mind rebasing so the tests complete successfully? Thanks for the PR!

**Comment by ubaumann** [comment]
Thanks for rebase it and thanks for all your time

**Pull request merged**